themes/clean-landing/layouts/partials/implementations.html

```diff
@@ -3,85 +3,27 @@
   <div class="content-wrapper">
     
     <div class="subtitle has-text-centered">
-      <h4>Implementations</h4>
+      <h4>Eco System</h4>
     </div>
 
     <div class="columns">
-      <div class="column is-quarter is-one-quarter-mobile has-text-left"><img src="images/crossplane.png" alt="Crossplane" /></div>
+      <div class="column is-one-fifths is-one-fifths-mobile has-text-left"><img src="images/crossplane.png" alt="Crossplane" /></div>
       
-      <div class="column is-three-quarters is-three-quarters-mobile">
-        <h1>Crossplane: The Kubernetes implementation of the Open Application Model</h1>
```

Comment:
Please keep the original narrative in this section.

Reply:
And we need to change another narrative above as we discussed.

Reply:
changed